internal/pkg/plugin/devlakeconfig/devlakeconfig.go

```diff
@@ -0,0 +1,109 @@
+package devlakeconfig
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/devstream-io/devstream/internal/pkg/plugininstaller"
+	"github.com/devstream-io/devstream/internal/pkg/statemanager"
+	"github.com/devstream-io/devstream/pkg/util/log"
+)
+
+var httpClient = &http.Client{
+	Timeout: 5 * time.Second,
+}
+
+func RenderAuthConfig(options plugininstaller.RawOptions) (plugininstaller.RawOptions, error) {
+	opts, err := NewOptions(options)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, p := range opts.Plugins {
+		for _, c := range p.Connections {
+			c.Token = c.Authx.Token
+			c.Username = c.Authx.Username
+			c.Password = c.Authx.Password
+			c.AppId = c.Authx.AppId
+			c.SecretKey = c.Authx.SecretKey
+		}
+	}
+
+	return opts.Encode()
+}
+
+func ApplyConfig(options plugininstaller.RawOptions) error {
+	opts, err := NewOptions(options)
+	if err != nil {
+		return err
+	}
+
+	for _, p := range opts.Plugins {
+		log.Infof("Got DevLake plugin config: %s. Connections: ", p.Name)
```

Comment:
missing `Connections: balabala`

Reply:
![image](https://user-images.githubusercontent.com/18692628/195226848-deefa341-e747-4a7d-a170-02113be1d11f.png)

The connection logs is followed below. @iyear cc @warren830